db/rdbms/src/main/java/io/camunda/db/rdbms/read/service/VariableReader.java

```diff
@@ -21,4 +29,13 @@ public VariableReader(final VariableMapper variableMapper) {
   public VariableDbModel findOne(final Long key) {
```

Comment:
❌  We should use VariableEntity here instead of VariableDbModel to be consistent with search method. 
🔧  Also in other implementations I have changed to Optional<> since it is a _findOne_ and not a _getOne_.